git_machete/git_operations.py

```diff
@@ -452,6 +455,9 @@ def get_tree_hash_by_commit_hash(self, commit_hash: FullCommitHash) -> Optional[
     def is_full_hash(revision: AnyRevision) -> Optional[Match[str]]:
         return re.match("^[0-9a-f]{40}$", revision)
 
+    def is_commit_present_in_repository(self, revision: AnyRevision) -> bool:
+        return True if self._run_git("rev-parse", f"--verify --quiet {revision}" + "^{commit}", allow_non_zero=True) == 0 else False
```

Comment:
Hmmm shouldn't each argument be in a separate string?
```suggestion
        return self._run_git("rev-parse", "--verify", "--quiet", revision + "^{commit}", allow_non_zero=True) == 0
```

Reply:
I'd actually be surprised if it worked in the current shape 🤔